src/app/core/services/authorization.service.ts

```diff
@@ -45,7 +58,7 @@ export class AuthorizationService {
   getUserInfo(): Observable<User> {
```

Comment:
Make this method private, since you are sharing `userData `via observable for your components.